src/material/autocomplete/testing/shared.spec.ts

```diff
@@ -1,84 +1,62 @@
+import {OverlayContainer} from '@angular/cdk/overlay';
 import {HarnessLoader} from '@angular/cdk/testing';
 import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
-import {Component, Type} from '@angular/core';
-import {ComponentFixture, TestBed, inject} from '@angular/core/testing';
+import {Component} from '@angular/core';
+import {ComponentFixture, inject, TestBed} from '@angular/core/testing';
 import {MatAutocompleteModule} from '@angular/material/autocomplete';
-import {OverlayContainer} from '@angular/cdk/overlay';
-import {MatAutocompleteModule as MatMdcAutocompleteModule} from '../index';
-import {MatAutocompleteHarness} from './autocomplete-harness';
-import {MatAutocompleteHarness as MatMdcAutocompleteHarness} from './mdc-autocomplete-harness';
-
-let fixture: ComponentFixture<AutocompleteHarnessTest>;
-let loader: HarnessLoader;
-let harness: typeof MatAutocompleteHarness;
-let overlayContainer: OverlayContainer;
-
-describe('MatAutocompleteHarness', () => {
-  describe('non-MDC-based', () => {
-    beforeEach(async () => {
-      await prepareTests(MatAutocompleteModule, AutocompleteHarnessTest);
-      harness = MatAutocompleteHarness;
-    });
-
-    runTests();
-  });
-
-  describe('MDC-based', () => {
-    beforeEach(async () => {
-      await prepareTests(MatMdcAutocompleteModule, AutocompleteHarnessTest);
-      // Public APIs are the same as MatAutocompleteHarness, but cast
-      // is necessary because of different private fields.
-      harness = MatMdcAutocompleteHarness as any;
-    });
-
-    // TODO: enable after MDC autocomplete is implemented
-    // runTests();
+import {MatAutocompleteHarness} from '@angular/material/autocomplete/testing';
+
+/**
+ * Function that can be used to run the shared autocomplete harness tests for either the non-MDC or
+ * MDC based checkbox harness.
+ */
+export function runTests(
```

Comment:
Should we name this `runHarnessTests`? Nothing from the package import implies that these are tests using the harnesses (i.e. `@angular/material/autocomplete/testing`)